"""resolvendo bug na tabela noticia

Revision ID: 99e51675544a
Revises: eacf52c4c1e1
Create Date: 2021-12-07 15:41:24.850798

"""
from alembic import op
import sqlalchemy as sa
from sqlalchemy.dialects import mysql

# revision identifiers, used by Alembic.
revision = '99e51675544a'
down_revision = 'eacf52c4c1e1'
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('noticia', schema=None) as batch_op:
        batch_op.add_column(sa.Column('img_noot', sa.String(length=100), nullable=True))
        batch_op.drop_column('img_not')

    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('noticia', schema=None) as batch_op:
        batch_op.add_column(sa.Column('img_not', mysql.VARCHAR(length=100), nullable=True))
        batch_op.drop_column('img_noot')

    # ### end Alembic commands ###
